README.md

```diff
@@ -1,32 +1,32 @@
-# MEMEBLAST
+# BLAST
+Persistent social image sharing feed for the whole world.
 
-## Komme i gang
+## Installing
 
-virtualenv memeblast && cd memeblast && source bin/activate
-git clone git@bitbucket.org:sigveseb/memeblast.git && cd memeblast
-cp memeblast/memeblast/settings.py.example memeblast/memeblast/settings.py
-pip install -r requirements_dev.txt
-python manage.py syncdb --migrate
+virtualenv memeblast && cd blast && source bin/activate 
```

Comment:
Might be, I just copied it exactly as it was.



Péter H. Gombos

On Mon, Mar 23, 2015 at 9:53 PM, Stian Jensen <notifications@github.com>
wrote:

>>  
>> -virtualenv memeblast && cd memeblast && source bin/activate
>> -git clone git@bitbucket.org:sigveseb/memeblast.git && cd memeblast
>> -cp memeblast/memeblast/settings.py.example memeblast/memeblast/settings.py
>> -pip install -r requirements_dev.txt
>> -python manage.py syncdb --migrate
>> +virtualenv memeblast && cd blast && source bin/activate 
> Isn't cloning before creating the virtualenv cooler, btw? This looks old...
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ninjadev/blast-server/pull/10/files#r26978791